server/train.py

```diff
@@ -56,16 +57,14 @@ def __init__(self, x, y, nickname, color, handle_train_death, tick_rate):
             "score": True,
             "color": True,
             "alive": True,
-            "speed": True,
-            "speed_boost": True,
-            "boost_cooldown": True,
+            "boost_cooldown_active": True
         }
         self.client_logger = logging.getLogger("client.train")
         # Speed boost properties
         self.speed_boost_active = False
         self.speed_boost_timer = 0
         self.boost_cooldown_active = False
-        self.boost_cooldown_timer = 0
+        self.start_cooldown_time = 0
```

Comment:
All the units should be in ticks (or we should aim to slowly get there). Agent.py should know the current room's tick_count.